README.md

```diff
@@ -240,3 +243,21 @@ describe('ComponentToTest', () => {
   }
 }
 ```
+
+
+## Release cycle:
+
+### Pre-releases
+
+To create a pre-release one can push/merge changes to branch next. This triggers actions to automatically create a pre-release.
+you can use the current pre-release by using `@next` as version in your project package.json.
+
+### Releases
+
+1. Merge your branch/changes into main-branch
+2. bump version in package.json with `npm version [<newversion> | major | minor | patch | premajor | preminor | prepatch | prerelease [--preid=<prerelease-id>]`
+3. push bumped version to main-branch with `git push --tags`
+4. create a new release with release notes from newly created tag on github
+5. The new release will trigger github-actions for automatically publishing to npm
+
+TODO: We should invest some time to optimize release handling
```

Comment:
This shouldn't be part of the README, let's create an issue for it. But like discussed today: what should be the goal? If the current way is documented and works - it should be okay. Let's rather invest in propert end-to-end tests for upgrades and refactorings.

Reply:
Got it. Created an unrefined issue for end to end tests